frontend/src/components/pages/Calendar/CustomizedCalendar.tsx

```diff
@@ -1,20 +1,43 @@
 import "./styles.css";
-import React, { useCallback, useMemo, useState } from "react";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
 import moment from "moment";
 import { Text, Box, Flex, Icon, IconButton, Select } from "@chakra-ui/react";
 import { FaChevronLeft, FaChevronRight } from "react-icons/fa6";
-import { Views, DateRange } from "react-big-calendar";
+import { Views, DateRange, Event } from "react-big-calendar";
 
 import Calendar from "../Calendar";
-import events from "./eventSeed";
+import ServiceRequestAPIClient from "../../../APIClients/ServiceRequestAPIClient";
+import { ServiceRequest, ServiceRequestType } from "../../../types/ServiceRequestTypes";
 
 type Keys = keyof typeof Views;
 
 export default function CustomizedCalendar() {
+  const [shifts, setShifts] = useState<Event[]>([]);
   const [selectedRequestType, setSelectedRequestType] = useState('');
   const [view, setView] = useState<(typeof Views)[Keys]>(Views.MONTH);
   const [date, setDate] = useState<Date>(moment(new Date()).toDate());
 
+  useEffect(() => {
+    const getAndTransformRequests = async () => {
+      const serviceRequests: ServiceRequest[] = await ServiceRequestAPIClient.get();
+      const events: Event[] = [];
+      serviceRequests.forEach((shift) => {
+        if (!shift.shiftTime || !shift.shiftEndTime) return;
+        events.push({
+          title: shift.requestName,
+          start: new Date(shift.shiftTime),
+          end: new Date(shift.shiftEndTime),
```

Comment:
Just check we're not using UTC, and convert to localtime using Date object

Reply:
Good point! I double checked and found that JavaScript converts the UTC timezone to local timezone. We should be good since MongoDB stores times in UTC by default.

E.g. 2024-02-16T17:37:43.068+00:00

<img width="510" alt="Screenshot 2024-06-11 at 6 51 14 PM" src="https://github.com/uwblueprint/abtc/assets/88808428/1c8952f2-c3cb-44ba-8c9c-b47330619a2b">

<img width="1438" alt="Screenshot 2024-06-11 at 6 48 28 PM" src="https://github.com/uwblueprint/abtc/assets/88808428/de0e34ee-07e4-4dbc-957b-7ed729501687">